CreateBootstrappedMSBuild.cmd

```diff
@@ -1,13 +1,12 @@
 :: Usage:
-:: BuildAndCopy
+:: Rebuild the repo and copy to bin\bootstrap. Includes required MSBuild
+:: companion files copied from your local machine.
 :: 
-:: Example: BuildAndCopy.cmd
+:: Example: CreateBootstrappedMSBuild.cmd <AdditionalBuildCommand>
```

Comment:
Pretty sure this isn't used in xplat and won't work. But that'll get cleaned up in #731.